docs/source/index.rst

```diff
@@ -47,103 +47,131 @@ Contents
    developers
    about
 
-.. panels::
-    :card: + intro-card text-center
+.. grid:: 1 2 2 2
+    :gutter: 3
 
-    ---
+    .. grid-item-card::
+        :text-align: center
 
-    Get Started
-    ^^^^^^^^^^^
+        Get Started
 
-    Get started using ``sktime`` quickly.
+        ^^^
 
-    +++
+        Get started using ``sktime`` quickly.
 
-    .. link-button:: get_started
-            :type: ref
-            :text: Get Started
-            :classes: btn-block btn-secondary stretched-link
+        +++
 
-    ---
+        .. button-ref:: get_started
+            :color: primary
+            :click-parent:
+            :expand:
 
-    User Guide
-    ^^^^^
+            Get Started
 
-    Find user documentation.
+    .. grid-item-card::
+        :text-align: center
 
-    +++
+        User Guide
```

Comment:
This is really pointing to the broader category of "User Documentation". Could consider changing these labels to "User Documentation" from "User Guide". But this PR does successfully convert to sphinx-design and retain the original label.